js/translation/i18n.js

```diff
@@ -16,7 +16,16 @@ const setLanguage = function(lng) {
     loadedTranslations = JSON.parse(fs.readFileSync(path.join(__dirname, '..', '..', 'locale', language + '.json'), 'utf8'));
 };
 
+/**
+ * Returns the current locale
+ * @return {*|string}
+ */
+const getLanguage = function() {
+    return language || 'en-US';
```

Comment:
Can't we default it to the system locale setting instead?

Reply:
the `language` variable is set through either user config / system locale. This is a safety check in case `language` is undefined, we fallback to the default language i.e. `en-US`